### ISSUE TITLE
upload after form submit

### ISSUE THREAD

**Issue by luckdragon** [blueimp/jQuery-File-Upload#805]
I implemented it exactly as you show on your page (verbatim) and it returns an error, it sees that the $_FILES array exists, but is blank, so, I modified it and set $uploads to null forcefully on the post directive, and now it just returns [] so how do I get the form to recognize that it's supposed to start the upload after submitting the data?  is there a way to do something like:

onSubmit ="upload Files" ?

basically, what I'm trying to do is have a regular form, and when they click submit on the regular form, it uploads the files from the jQuery-File-Upload box, and then submits the form
